crates/uv/tests/pip_compile.rs

```diff
@@ -5458,7 +5458,10 @@ fn unsupported_scheme() -> Result<()> {
     ----- stdout -----
 
     ----- stderr -----
-    error: Unsupported URL prefix `bzr` in URL: `bzr+https://example.com/anyio` (Bazaar is not supported)
+    error: Couldn't parse requirement in `requirements.in` at position 0
+      Caused by: Unsupported URL prefix `bzr` in URL: `bzr+https://example.com/anyio` (Bazaar is not supported)
+    anyio @ bzr+https://example.com/anyio
```

Comment:
My only concern here is: isn't this a valid PEP 508 URL? It's just that uv doesn't support it?

Reply:
Oh I see, I guess this only appears because we parameterize the type on `VerbatimParsedUrl` instead of `VerbatimUrl`?

Reply:
Yes i've pushed to parsing down. If we decide wanted to support preinstalled urls we don't support, we can add an `Unknown` variant to `VerbatimParsedUrl`, moving some of the error handling back later again.